ext/lib/site/home-proyectos/component.js

```diff
@@ -50,55 +55,72 @@ export class HomeProyectos extends Component {
-    //   credentials: 'include'
-    // })
-      .then((res) => res.json())
-      .then((res) => {
-        this.setState({
-          barrio: initialFilters.barrio ? [ initialFilters.barrio ] : [],
-          // IMPORTANTE LEER
-          // This filters should be applied if seguimiento stage is active only
-          // state: defaultValues.seguimiento.state,
-          state: ['preparacion', 'compra', 'ejecucion', 'finalizado'],
-          anio: defaultValues.seguimiento.anio,
-          stage: 'seguimiento',
-          // IMPORTANTE LEER
-          // El siguiente se activa para mostrar los proyectos a votar
-          // state: this.props.location.query.stage === 'seguimiento' ? ['no-ganador', 'preparacion', 'compra', 'ejecucion', 'finalizado'] : ['factible'],
-          // IMPORTANTE LEER
-          // Este se activa cuando paso la votacion y se muestran los gandores
-          // state: this.props.location.query.stage === 'seguimiento' ? ['no-ganador', 'preparacion', 'compra', 'ejecucion', 'finalizado'] : ['preparacion', 'compra', 'ejecucion', 'finalizado'] ,
-          // anio: this.props.location.query.stage === 'seguimiento' ? ['2018', '2019','2020','2021','2022'] : ['2023'],
-          // IMPORTANTE LEER
-          // Activar cuando se pasa a votacion
-          // stage: this.props.location.query.stage === 'seguimiento' ? 'seguimiento' : 'votacion',
-          topics: res.results.topics,
-          page: res.pagination.page,
-          noMore: res.results.topics.length < 20
-        }, () => {
-          this.firstSeeMore()
-        })
+    const getInfo = async () => {
+      await forumStore.findOneByName('proyectos').then((forum) => {
+        this.setState({ forumConfig: forum.config })
       })
-      .catch((err) => console.error(err))
+
+      let initialFilters = {}
+      if (this.props.location.query.barrio) initialFilters.barrio = this.props.location.query.barrio
+      if (this.props.location.query.tag) initialFilters.tags = this.props.location.query.tag
+
+      if (this.state.forumConfig.preVotacion) {
+        // pre-votacion seccion seguimiento: muestra, 'preparacion', 'compra', 'ejecucion', 'finalizado' de votaciones pasadas
+        // pre-votacion seccion votacion: muestra, 'factibles' de año de votacion
+        initialFilters.state = this.props.location.query.stage === 'seguimiento' ? 'no-ganador,preparacion,compra,ejecucion,finalizado' : 'factible'
+        initialFilters.anio = this.props.location.query.stage === 'seguimiento' ? '2018,2019,2020,2021,2022' : '2023'
+      } else if (this.state.forumConfig.votacionFinalizada) {
+        // votacionFinalizada seccion seguimiento: muestra, 'preparacion', 'compra', 'ejecucion', 'finalizado' de votaciones pasadas
+        // votacionFinalizada seccion ganadores: muestra, 'no-ganador', 'preparacion', 'compra', 'ejecucion', 'finalizado' de año de votacion
+        initialFilters.anio = '2018,2019,2020,2021,2022'
```

Comment:
Esto deberia ser similar a 

`this.props.location.query.stage === 'seguimiento' ? '2018,2019,2020,2021,2022' : '2023'`

O sea... 

`this.props.location.query.stage === 'seguimiento' ? previousYears.join(',') : currentYear`